platform-dom/src/main/java/com/softicar/platform/dom/input/IDomTextualInput.java

```diff
@@ -88,4 +88,28 @@ default void select() {
 
 		getDomEngine().select(this);
 	}
+
+	// -------------------------------- read-only -------------------------------- //
+
+	/**
+	 * Makes this {@link IDomTextualInput} read-only.
+	 * <p>
+	 * In contrast to {@link #setEnabled(boolean)}, the user can still see and
+	 * copy the text of this {@link IDomTextualInput}.
```

Comment:
I tested with Firefox. You _can_ copy the text of a disabled input field, too. So this Javadoc comment is not right.

Reply:
interesting

Reply:
I removed this comment.